Sources/AEPOptimize/Event+Optimize.swift

```diff
@@ -65,7 +65,19 @@ extension Event {
                             type: EventType.optimize,
                             source: EventSource.responseContent,
                             data: [
-                                OptimizeConstants.EventDataKeys.RESPONSE_ERROR: error.rawValue
+                                OptimizeConstants.EventDataKeys.RESPONSE_ERROR: error
```

Comment:
nit: If `createErrorResponseEvent(_ error: AEPError)` isn't being used anymore, can we remove it?

Reply:
since we return AEPError in getProposition api, we are using this function. If we return AEPOptimizeError instead, we can remove this function.

Reply:
Discussed offline to return `AEPOptimizeError` in getPropositions as well for API consistency